src/Pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,25 @@
WZKtzmrLYcrH9YXhsVrm1iOce9MNa4BtHooGyVbNJkSgzsYJ0SVHAx14wGPooWIUbSbCAWmhqOZc5Bdl+Nsz4D3kVTHUGf4tRw25EHeOqAQCRsBJN84J2kxwpU1oIe8w7x4Lv8AFj6Xmcwwmviw5d1OeXH8XtX7P8SK3xGKgFNQKPwqwYHx7tr/AK3288i7C6Qag2vYmmbK9s9UN8rW5EH1G8T1mnVVgGBBBFwQciOM3uOuo5st27SQbGbIRcQXGsLRFD8JshEFwbC0JvAU6KNuIoE+RRiH/G39RklPEuDkzed/eDqJKhKsDbMG+ch0rqhi6oFyoPO9jLXQWIaq4SwBZlVeFybZ+coWxZ1SCLMfY75eaLoauoAdU3Fm4G4z84up7E29v0do40UVF2Lv3k7yeZnNIYw0gLnvHYOW9jymdxPbJittZFNrEqCWJ3kZm3l4zOYzSOv8pNm232m19vXWHlDlPiZhfeS/xenCx2l+uSjoIRonHM7qFFmuBlvW/e8hc+EyuGVnYKoJJIAA2knYBPTdAaGXDrc2NRh3juH8K8ue/wAgKk2eVmM0ynaXRLVcbh1YXVlueYplnIPLYP8AVPLdOhviuxG2o+fPWuR6z3ntBpLD0GR6rqrDXUDNm1XXMhVubXVRe2+eMdoq1BxiSj/95KlMFWGur64cC4ysXU57Qs0ZSlSxWuignvJ3eq/unwzHlETKvDm6aw2ixHlmPWGUawYAj/5ynNlHRKe1+NvCDvSP428x9BJyYNiKGtmGKnkfcSIZuo42Pfr9mPSsw+ZbDiMx48BBQlYfvBuoHuCIZRLH5gB0N/pKsKCAZ28jRbbI68jRn3lpQK0ENV/mIsq789ijmfQQHD6iL8WobKD3Rvc8hv8AvdKjG6Ras5dslAOqu5R+ZtmY5jcrob4hcZUeq7ucyO81tgFwvgLsq+UnwKd4TW/sx0QuJGMVv3qKoDwNRiwbwZFPhM7hqBV9VhZgSCDuINiPAgzsxmunPvdb/s9gzTxCKANSvSDgWy1XTWK25MhHS022E0c1PJCwXbq3uvhfMSo0fQ7uj23hSvhYEexmyEKm5K/4TwevTYbZcQXG7JIlA0KbHZJxReEYId2EwFqv+E8UsJ2BbfJGAp6z57s4fjcOLB7Zi1+YvA9FN3/D6iXNcjVN+BmVvbonpX12BdLcQfAkW+svKbbOv0MzGABLryz8vsS+NUKLnwhZ8EqzV5Y4XBu1ixRFP71R0pDqNcgsOYBmSfGte99W2yxtbxip194MrHD+py8n8e69ktBU6SitrpVYjushDIo36rbzzy4cbwdsO2C4a9KkQ1a2Z2rTB2X4ty3bTwPl2gu1NfC6/wAJwA6kMCLqDawcD8Y4+d5R43Hs5JJJJJLEm5JOZJO8zWYsr3ex2kdJs7MzMXYm5JNyTzMp6mKzztwkNWtukBhaF5o1u542kL1/hufwnM/mJzRD91hz9xJcRR1mH8pHqPqZhf2rad4wargi4j5UUaxpnVbZ7cxyvLFKgIuDcTO46XLtJHSPXkVXEKu0/megi0exBaDVMaiHPvEfug2v1O4evvK/EY4m4HdGQvvz9vCQ0l9h65mVMf6i5fwVicS9Q6zm5+VQMgo4KNw/KNxoCLbebX9/aTYanc6x3ZDrvlfpGrdvM+ez2mmPvScvT0P9lGmKeHr1KdRtX4qoFY/KGUtYMd19e1+I5x/arCBdI1lUfMyMAPxOisR1LE+c8+w2IORP3xmz7NaapriBiMSXqFQCtgrFmUBVLFiMlUA7ySBNp/WUexUMBqrhl/8AEAD4UynuZZzEf8QqR2UW/wBTKPYGWOA7ZUHIDBkJ3nvL5jMeUNUtVpoNjdkmSoGAIIIOYINwRxBEgxxykiHYP5YRBsGe7CbwFKKKKBPkjCGzjqR7y4xDgIdbZ78oT2n7PDAfCRn167qztawRFvqgAEazEnW7xI+U5TOVKzNtJMm4XbXnBeHqKl2NtY7tw5RtTFFjxP3sEDvCEsoz2ypjIi5WuhWbb6/lJ0QDbnBWxB3ZR9E/vE/e+UQmpVsOZgrVJG9S5vGhoA8GNZpy84TALPQ75sOQPkf1lq20dfp+dpS6KHevxuPr9JcOcumflnOfP9m+H6m4jDhhK57obZg8pbSJ0VxmIpVWK9a72+Y+kgW5a/HP6wmphWGS7PpI1pEZkb/0+krcT2Z8Ld4ydEzsN5kyYbefKSURdidw7o+v3zi2JilburluEptJUyNVv4c+t7/WXGIFwBxIHhtPtFVpBhYjKLHLVPLHcZ+g8tMLX3eUrcThjTbiNx+nWdSpadMrCzS5XFG9jLHDaQI3ygqPrLrDdO0q8qUbendmu1TUCAe/TJzW+z+JeB9/Weh/5jTrIGpsGG/iDwYbQZ8+4fFkS+0XpdkYFWKnkdvI8RyMLjKb3DAN3YTeZzsnppa6FTYOouRuI/EPqN00cizSaV4py87ET5Mx2Leq5d3Z2O1mYsx8TnB4xY68o3VbMef5TrNfORoY6ALfaSu+VhskC8Y6IE5yjhI22iOjB14gLxsIwiXN+HvFbqHJsdhhqleo9cvrLPaJXOuqR4H1h4M58m8ORrgdJChs5G43I9L/AFkifU+8jqjNTwPuLflEaeR6tzfhs67z9848GcERuVnsCfu+6dopYAffORtmwHDP8vr5SQR/ARF2HIH1+zJJHzndaIGV0DCxFxM+41WI4EjymgczPYj52/mPvNvGz8iVH7rdP0iR4OW9Y8NNWIxKkOw9aVCvCKdSOU5W+7J6VNOtTa+xgDzUmzDyJntd5826Nxeqb8CD6z0Sl+0WszX1KVuFnOXXWjvYs29OvFMF/wAQD/4V/rP5RSeI4vAFjmiiiI1I6KKH0GrHxRRg3eI6KKIyMP0d9fpOxScvR4exOJ3dIaIopjW8OE4YopJuiKKKIka7T4fWSCKKUIU5FFFAZUlBifnbqfeKKa4I8npEY8RRTZi6snSKKAGYXf4Swwe0RRSoqLOKKKWb/9k="
```

Comment:
replace this image, check this article on how to add images to react: https://create-react-app.dev/docs/adding-images-fonts-and-files/